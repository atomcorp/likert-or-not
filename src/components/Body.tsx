import * as React from 'react';
import css from '../index.css';

import LikertOrNot from '../index.d';

const Body = (props: LikertOrNot.RowsProps) => (
  <tbody className={css.rows}>{props.children}</tbody>
);

export default Body;
