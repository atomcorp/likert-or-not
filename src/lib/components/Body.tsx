import * as React from 'react';
import css from '../Matrix.css';

import {
  RowsProps,
  // DomRowType,
  // ReturnValueProps,
  // Radio,
  // IsCheckProps,
} from '../definitions';

const Body = (props: RowsProps) => (
  <tbody className={css.rows}>{props.children}</tbody>
);

export default Body;
