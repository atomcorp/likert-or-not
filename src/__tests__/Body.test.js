import React from 'react';
import Body from '../components/Body';
import renderer from 'react-test-renderer';

test('Body prints text in tbody', () => {
  const component = renderer.create(<Body><div>The Body node</div></Body>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
