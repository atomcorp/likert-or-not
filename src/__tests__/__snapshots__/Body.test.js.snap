// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body prints text in tbody 1`] = `
<tbody
  className={undefined}
>
  <div>
    The Body node
  </div>
</tbody>
`;
